android/src/com/frostwire/android/core/Constants.java

```diff
@@ -180,8 +180,8 @@ private Constants() {
     public static final String SOCIAL_URL_TWITTER_PAGE = "https://twitter.com/frostwire";
     public static final String SOCIAL_URL_REDDIT_PAGE = "https://reddit.com/r/frostwire";
 
-    public static final String VPN_LEARN_MORE_URL = "http://www.frostwire.com/vpn";
+    public static final String VPN_LEARN_MORE_URL = "https://www.expressvpn.com/internet-security/private-vpn/buy?a_aid=frostwire";
```

Comment:
if you need another URL to directly link to expressvpn, then use another variable.